plugin/src/generators/create-community/generator.ts

```diff
@@ -0,0 +1,66 @@
+import { logger, readJson, Tree, updateJson } from '@nx/devkit';
+import { CreateCommunityGeneratorSchema } from './schema';
+import { exit } from 'node:process';
+import { Community } from '../models/community';
+
+const COMMUNITIES_PATH = 'angular-hub/src/public/assets/data/community.json';
+
+export async function createCommunityGenerator(
+  tree: Tree,
+  options: CreateCommunityGeneratorSchema,
+) {
+  const { name, type } = options;
+
+  const packageJson = readJson(tree, 'package.json');
+  const angularCorePackage = packageJson['dependencies']['@angular/core'];
```

Comment:
angular/core check to be removed as the generator is always used in the workspace.